css/mobile_first.css

```diff
@@ -116,22 +135,392 @@ body {
   margin-top: 5%;
 }
 
+.icon-2 {
+  list-style-type: none;
+  margin-right: 10%;
+}
+
 h3 {
   margin-top: 12px;
   margin-bottom: 0;
   font-weight: bold;
-  font-size: 32px;
+  font-size: 18px;
+  color: var(--gris);
+  line-height: 25px;
+}
+
+#h3-lang {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 18px;
   color: var(--gris);
-  line-height: 44px;
+  line-height: 25px;
 }
 
-footer h3 {
-  font-style: normal;
+#my-recent-works {
```

Comment:
- [ ] Kindly adjust the margin at the top of the works section to look visually appealing and give a better user experience. This is also important in sticking to requirements 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/227675397-42f714f2-beae-4e87-a047-fd9b31858ba2.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/227675422-96a6da0e-a729-4c56-baaa-f4bc659cc1d8.png)